server/container_create_linux.go

```diff
@@ -553,8 +553,13 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrIface.Contai
 	created := time.Now()
 	spp := containerConfig.Linux.SecurityContext.SeccompProfilePath
 	if !ctr.Privileged() {
-		if err := s.setupSeccomp(ctx, specgen, spp); err != nil {
-			return nil, err
+		if err := s.setupSeccompFromProfilePath(ctx, specgen, spp); err != nil {
+			return nil, errors.Wrap(err, "setup seccomp from profile path")
+		}
+		if err := s.setupSeccomp(
+			ctx, specgen, securityContext.Seccomp,
+		); err != nil {
+			return nil, errors.Wrap(err, "setup seccomp")
```

Comment:
what would you think about moving some of this into pkg/container? we could pass in the server related information, and have it be the editor of the spec?

Reply:
We would need the same logic for the sandbox as well as the container. I could refactor the code parts into `internal/config/seccomp/seccomp.go`, which then could be used from both. WDYT?

Reply:
eventually we'll need to combine functionality between sandbox and container packages, as an advantage of factoring them out is to be able to share common code. putting it in `internal/config/seccomp/seccomp` is a good strategy for now though

Reply:
Alright, I'll do that. Yeah maybe Sandbox and Container can share a common interface somehow.

Reply:
that's my dream at least :) one day...